go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -4183,6 +4216,54 @@ func StartServer(s *grpc.Server, ts *topo.Server) {
 	vtctlservicepb.RegisterVtctldServer(s, NewVtctldServer(ts))
 }
 
+// getTopologyCell is a helper method that returns a topology cell given its path.
+func (s *VtctldServer) getTopologyCell(ctx context.Context, cellPath string) (*vtctldatapb.TopologyCell, error) {
+	// extract cell and relative path
+	parts := strings.Split(cellPath, "/")
+	if parts[0] != "" || len(parts) < 2 {
+		err := vterrors.Errorf(vtrpc.Code_INVALID_ARGUMENT, "invalid path: %s", cellPath)
+		return nil, err
+	}
+	cell := parts[1]
+	relativePath := cellPath[len(cell)+1:]
+	topoCell := vtctldatapb.TopologyCell{Name: parts[len(parts)-1], Path: cellPath}
+
+	conn, err := s.ts.ConnForCell(ctx, cell)
+	if err != nil {
+		err := vterrors.Errorf(vtrpc.Code_UNAVAILABLE, "error fetching connection to cell %s: %v", cell, err)
+		return nil, err
+	}
+
+	data, _, dataErr := conn.Get(ctx, relativePath)
```

Comment:
We cannot early-return if this error is non-nil, because when the cell is a directory cell, `conn.Get` will return an error when "getting" data.

Instead, we return an error if both `conn.ListDir` and `conn.Get` both return errors.

Reply:
> @ajm188 I wasn't sure if we wanted another cmd like the topocat one instead of this one for the command line client. I'll go ahead and add this one though.

sounds good! i would have then argued if people can "just" use topocat for this functionality then so should we 😂

Reply:
uhhhhh, i have no idea why this got threaded on this particular comment, sorry about that!

Reply:
Oh I didn't see this because it got put in a weird place... I went ahead and added the command anyway!